### PR TITLE
Kitchen Sink: Build A Quiz App

### PR DESCRIPTION
**Pull request by arunavabasu-03**
**Please describe the changes this PR makes and why it should be merged:**
This PR aims to demonstrate  a quiz app  example using million . 
**Status**
Closes : https://github.com/aidenybai/million/issues/607

- [x] Code changes have been tested against prettier, or there are no code changes
- [x] I know how to update typings and have done so, or typings don't need updating

**Semantic versioning classification:**

- [ ] This PR changes the codebase
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
  - [ ] This PR changes the internal workings with no modifications to the external API (bug fixes, performance improvements)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: packages/kitchen-sink/src/examples/quiz.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  padding: 10px 20px;
+  border-radius: 5px;
+  cursor: pointer;
+  transition: background-color 0.3s;
+
+  &:hover {
+    background-color: #0056b3;
+  }
+`;
+
```
Comment (deeola) [inline packages/kitchen-sink/src/examples/quiz.tsx:53]: Can we move the styling to a different file please? It helps with the general readabilty.

**Comment by aidenybai** [comment]
Please run `pnpm run cleanup` @arunavabasu-03

**Review by tobySolutions — CHANGES_REQUESTED** [review]
Could you maybe try using the `<For/>` component to replace the map you have? @deeola? 
https://million.dev/docs/for

**Comment by arunavabasu-03** [comment]
> Could you maybe try using the `<For/>` component to replace the map you have? @deeola? https://million.dev/docs/for

it try to do that but it with radio select it is not working properly so done with **`map`** .

**Comment by arunavabasu-03** [comment]
> Please run `pnpm run cleanup` @arunavabasu-03

@aidenybai done cleanup !

**Comment by arunavabasu-03** [comment]
> > > Could you maybe try using the `<For/>` component to replace the map you have? @deeola? https://million.dev/docs/for
> > 
> > 
> > it try to do that but it with radio select it is not working properly so done with **`map`** .
> 
> Hmm, can you send a screen recording of this behaviour and your code so we can document it? Then you can revert back to map


code - 
```typescript 
 <For each={questions[currentQuestion].options}>
              {(option, index) => (
                <OptionLabel key={index}>
                  <OptionInput
                    type="radio"
                    value={option}
                    checked={selectedOption === option}
                    onChange={handleOptionChange}
                  />
                  {option}
                </OptionLabel>
              )}
            </For>
```

https://github.com/aidenybai/million/assets/73842340/5031c553-60ee-4213-b836-c6f8687f39e9

**Comment by tobySolutions** [comment]
> > > > Could you maybe try using the `<For/>` component to replace the map you have? @deeola? https://million.dev/docs/for
> > > 
> > > 
> > > it try to do that but it with radio select it is not working properly so done with **`map`** .
> > 
> > 
> > Hmm, can you send a screen recording of this behaviour and your code so we can document it? Then you can revert back to map
> 
> code -
> 
> ```ts
>  <For each={questions[currentQuestion].options}>
>               {(option, index) => (
>                 <OptionLabel key={index}>
>                   <OptionInput
>                     type="radio"
>                     value={option}
>                     checked={selectedOption === option}
>                     onChange={handleOptionChange}
>                   />
>                   {option}
>                 </OptionLabel>
>               )}
>             </For>
> ```
> 
>  million_for.mp4
Thanks for this

**Comment by tobySolutions** [comment]
Hey there @arunavabasu-03! This is what I have here:


https://github.com/aidenybai/million/assets/96334363/06ba02e5-3598-48e4-b77c-dbfde82815c6

**Comment by arunavabasu-03** [comment]
@tobySolutions I think it is because of your dark mode. Can you disable and check?

**Comment by tobySolutions** [comment]
Hey there @arunavabasu-03, how are you?? The below should fix your issues, check it out and make your fixes. Thanks!



https://github.com/aidenybai/million/assets/96334363/237a9eac-6dcc-4fcc-aae3-b0dd04020799

**Comment by tobySolutions** [comment]
Hey there @arunavabasu-03, we already have a quiz app on the sink, but, let's review your PR and get it merged shall we?

**Review by tobySolutions — APPROVED** [review]
LGTM!

**Comment by tobySolutions** [comment]
I think we still have the same issue. I'm gonna merge this for future reference by the way. I think this might be an issue with styled components.

**Comment by tobySolutions** [comment]
I'll be closing this for now. I'll need @aidenybai and the rest of the team to look into this though

**Pull request closed without merging**